features/admin.applications.v1/components/forms/inbound-oidc-form.tsx

```diff
@@ -435,6 +457,74 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         );
     }, [ application ]);
 
+    const fetchLocalClaims: () => void = useCallback(() => {
+        getAllLocalClaims(null)
+            .then((response: Claim[]) => {
+                setClaims(response);
+            })
+            .catch(() => {
+                dispatch(addAlert({
+                    description: t("claims:local.notifications.fetchLocalClaims.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("claims:local.notifications.fetchLocalClaims.genericError.message")
+                }));
+            });
+    }, []);
+
+    const fetchExternalClaims: () => void = useCallback(() => {
+        getAllExternalClaims(OIDCScopesManagementConstants.OIDC_ATTRIBUTE_ID, null)
+            .then((response: ExternalClaim[]) => {
+                setExternalClaims(response);
+            })
+            .catch(() => {
+                dispatch(addAlert({
+                    description: t("claims:external.notifications.fetchExternalClaims.genericError.description"),
+                    level: AlertLevels.ERROR,
+                    message: t("claims:external.notifications.fetchExternalClaims.genericError.message")
+                }));
+            });
+    }, []);
+
+    useEffect(() => {
+        fetchLocalClaims();
+        fetchExternalClaims();
+    }, []);
+
+
+    useEffect(() => {
+        if (claims.length && externalClaims.length) {
+            const updatedAttributes : ExternalClaim[] = externalClaims.map((externalClaim : ExternalClaim) => {
+                const matchedLocalClaim: Claim = claims.find((localClaim: Claim) =>
+                    localClaim.claimURI === externalClaim.mappedLocalClaimURI
+                );
+
+                if (matchedLocalClaim && matchedLocalClaim.displayName) {
```

Comment:
we can shorted this as follows.

```suggestion
                if (matchedLocalClaim?.displayName) {
```

Reply:
fixed